src/word.c

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
```

Comment:
The indentation in this file is a bit of a hot mess. Please pass everything through `astyle`.

Reply:
I think this is corrected now